Create flag to prioritize dequeuing from query components over tenant fairness (#9016)
grafana/mimir: docs/sources/mimir/configure/configuration-parameters/index.md

# CLI flag: -query-scheduler.use-multi-algorithm-query-queue
[use_multi_algorithm_query_queue: <boolean> | default = false]

> # (experimental) When enabled, the query scheduler will primarily prioritize
> # dequeuing fairly from queue components, and secondarily prioritize dequeuing
> # fairly across tenants. When disabled, tenant fairness is primarily
> # prioritized. `query-scheduler.use-multi-algorithm-query-queue` must be enabled
> # in order to use this flag.

Reviewer: ```suggestion
# (experimental) When enabled, the query scheduler primarily prioritizes
# dequeuing fairly from queue components and secondarily prioritizes dequeuing
# fairly across tenants. When disabled, the query scheduler primarily
# prioritizes tenant fairness. You must enable the `query-scheduler.use-multi-algorithm-query-queue` setting to use this flag.
```

Reviewer: We avoid using "will" in docs and instead always use the present tense.
